packages/ckb-sdk-core/src/index.ts

```diff
@@ -5,15 +5,30 @@ import { ArgumentRequired } from '@nervosnetwork/ckb-sdk-utils/lib/exceptions'
 import ECPair from '@nervosnetwork/ckb-sdk-utils/lib/ecpair'
 import * as utils from '@nervosnetwork/ckb-sdk-utils'
 
-import generateRawTransaction from './generateRawTransaction'
+import generateRawTransaction, { Cell, RawTransactionParamsBase } from './generateRawTransaction'
 
 import loadCells from './loadCells'
-import signWitnessGroup from './signWitnessGroup'
+import signWitnesses, { isMap } from './signWitnesses'
 
-type StructuredWitness = CKBComponents.WitnessArgs | CKBComponents.Witness
+
+type Key = string | ECPair
+
+interface RawTransactionParams extends RawTransactionParamsBase {
+  fromAddress: string
+  toAddress: string
+  capacity: bigint | string
+  cells?: Cell[]
+}
+
+interface ComplexRawTransactoinParams extends RawTransactionParamsBase {
```

Comment:
I think `ComplexRawTransactoinParams ` will be more generical and when sdk implements multi-sig, the parameters may be different from simple and complex parameters.

Reply:
It's an inner interface name, we can update it later if necessary.

Reply:
Get